integration-tests/services/node-module/tests/basic-query.exotest

```diff
@@ -0,0 +1,10 @@
+operation: |
+  query {
+    test_builtin
+  }
+response: |
+  {
+    "data": {
+      "test_builtin": "x64"
```

Comment:
This will fail for aarch64 Mac. Is there another method to call? If not, maybe implement "os_bits" to return last to characters of `os.arch()`, which will return "64" for all our current targets.

Reply:
changed to just check for successful API call